src/enum_variant_accessor.rs

```diff
@@ -459,10 +459,10 @@ pub fn impl_enum_accessor(input: DeriveInput) -> TokenStream {
     {
         match parse_attr(attr) {
             Ok(accessor) => {
-                if accessors.iter().any(|a| a.ident == accessor.ident) {
+                if accessors.iter().any(|a| a.alias == accessor.alias) {
                     return syn::Error::new(
-                        accessor.span,
-                        format!("Duplicate accessor {}", accessor.ident),
+                        accessor.alias.span(),
```

Comment:
`accessor.alias` will be either the name itself, or the alias if one is provided ; by taking its span, compilation error with highlight it in both cases